### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by chenyong001**
1、解决ant编译sass失败问题
2、切换为中文后，使用chat and sucrib组件时前端代码异常，双引号不匹配

### CODE REVIEW

**Comment by chenyong001** [comment]
ok

File: lams_build/build_base.xml [hunk 1]
```diff
@@ -372,7 +372,8 @@
 	<!-- ====================================================================== -->
 	<target name="_sass.do.compile">
         <apply dest="${basedir}/build/css" executable="${sass_exec_file}" failonerror="true" verbose="true">
-        	<arg value="--no-source-map"/>
+<!--        	<arg value="&#45;&#45;no-source-map"/>-->
+					<arg value="--sourcemap=none"/>
```
Comment (MarcinCieslak) [inline lams_build/build_base.xml:376]: Please see this commit for explanation what this form of the parameter is the way to go
https://code.lamsfoundation.org/fisheye/changelog/lams-github?cs=6aeea5d482f2cddbd3b75c4c722de6caf7ab633f

File: lams_central/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+测试1
```
Comment (MarcinCieslak) [inline lams_central/README.md:1]: This probably should be discarded, right?

File: lams_tool_scribe/web/pages/learning/scribe.jsp [hunk 3]
```diff
@@ -123,7 +123,8 @@
 			}
 		
 			function confirmForceComplete() {
-				return confirm("<fmt:message key='message.confirmForceComplete'/>");
+				<%--return confirm("<fmt:message key='message.confirmForceComplete'/>");--%>
+				return confirm('fmt:message key="message.confirmForceComplete"/>');
```
Comment (MarcinCieslak) [inline lams_tool_scribe/web/pages/learning/scribe.jsp:127]: I believe your code misses an opening "<". It should be
```
return confirm('<fmt:message key="message.confirmForceComplete"/>');
```
Can you explain in English why this change is required? Does it cause a problem when it gets translated to other languages?

**Review by MarcinCieslak — COMMENTED** [review]
Hi!
Thank you for your change propositions.
Unfortunately we can only take feedback in English...
Also see my comments to your changes.

**Comment by MarcinCieslak** [comment]
No comment from developer, closing pull request.
